v2/go.mod

```diff
@@ -1,5 +1,10 @@
 module github.com/nats-io/jwt/v2
 
-go 1.16
+go 1.18
 
-require github.com/nats-io/nkeys v0.3.0
+require (
+	github.com/nats-io/jwt v1.2.2
```

Comment:
I think this will make vulnerability checks fail on nats-io/jwt v2 because v1 interpreted as a dependency of v2 not just for tests.

Reply:
ok will revisit.

Reply:
reverted.